cluster-scope/README.md

```diff
@@ -34,6 +37,11 @@ The directories and files in this structure fulfill the following purpose:
 - `kustomization.yaml` : Kustomization file including a single resource, which is a sibling file within the folder. Can pull in `components` for additional bootstrapping.
 - `<resource_kind_singular>.yaml` : **Single** resource manifest with a `.spec` that can be deployed to all the clusters.
 
+#### bundles/
+Resource Bundling was introduced in this [PR](https://github.com/operate-first/apps/pull/1552). And is discussed more extensively [here](https://github.com/operate-first/SRE/issues/442)
```

Comment:
I think we should just explain briefly when/how users should bundle resources instead of linking to threads (otherwise they would have to read entire discussions to understand).

Reply:
done